tests/e2e/json_test.go

```diff
@@ -55,8 +55,13 @@ var _ = Describe("odojsonoutput", func() {
 		It("should be able to list url", func() {
 			actual := runCmdShouldPass("odo url list -o json")
 			url := runCmdShouldPass("oc get routes myurl-myapp -o jsonpath={.spec.host}")
-			desired := fmt.Sprintf(`{"kind":"List","apiVersion":"odo.openshift.io/v1alpha1","metadata":{},"items":[{"kind":"url","apiVersion":"odo.openshift.io/v1alpha1","metadata":{"name":"myurl","creationTimestamp":null},"spec":{"path":"%s","port":8080}}]}`, url)
+			desired := fmt.Sprintf(`{"kind":"List","apiVersion":"odo.openshift.io/v1alpha1","metadata":{},"items":[{"kind":"url","apiVersion":"odo.openshift.io/v1alpha1","metadata":{"name":"myurl","creationTimestamp":null},"spec":{"path":"%s","protocol":"http","port":8080}}]}`, url)
```

Comment:
going over the 80 character length, if we can make this into more than one lines

Reply:
Character length seems ```OK``` to me as ```golint``` does not report anything in https://github.com/redhat-developer/odo/pull/1330#issuecomment-463571158